common/simple_junit/include/simple_junit/junit5.hpp

```diff
@@ -287,10 +298,20 @@ struct SimpleTestSuite : private std::unordered_map<xs::string, SimpleTestCase>
 
     current_node.append_attribute("name") = testsuite.name.c_str();
 
+    size_t tests = 0;
+    size_t failures = 0;
+    size_t pass = 0;
+    size_t errors = 0;
```

Comment:
`size_t` => `std::size_t`

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/529/commits/53fee2d8a5fb29abe5efb9a352b175f8a2885e20